docs/quick-start/environment.mdx

```diff
@@ -15,7 +15,10 @@ In the first terminal, run a local network:
 yarn chain
 ```
 
-This command starts a local Ethereum network using Hardhat. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in `hardhat.config.ts`.
+This command starts a local Ethereum network using Hardhat or Foundry, depending on which one you selected in the CLI. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in:
+
+- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.
+- `packages/foundry/foundry.toml` if you have Foundry as solidity framework.
```

Comment:
Should we do it like below to keep the same formatting:

- Hardhat => `packages/hardhat/hardhat.config.ts`
- Foundry => `packages/foundry/foundry.toml`